docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -1,29 +1,51 @@
 [[searchable-snapshots]]
 == {search-snaps-cap}
 
-{search-snaps-cap} let you reduce your operating costs by using
-<<snapshot-restore, snapshots>> for resiliency rather than maintaining
-<<scalability,replica shards>> within a cluster. When you mount an index from a
-snapshot as a {search-snap}, {es} copies the index shards to local storage
-within the cluster. This ensures that search performance is comparable to
-searching any other index, and minimizes the need to access the snapshot
-repository. Should a node fail, shards of a {search-snap} index are
-automatically recovered from the snapshot repository.
-
-This can result in significant cost savings for less frequently searched data.
-With {search-snaps}, you no longer need an extra index shard copy to avoid data
-loss, potentially halving the node local storage capacity necessary for
-searching that data. Because {search-snaps} rely on the same snapshot mechanism
-you use for backups, they have a minimal impact on your snapshot repository
-storage costs.
+{search-snaps-cap} let you use <<snapshot-restore,snapshots>> to search and
+store infrequently-accessed, read-only data. The <<cold-tier,cold>> and
+<<frozen-tier,frozen>> data tiers use {search-snaps} to reduce your storage and
+operating costs.
+
+{search-snaps-cap} eliminate the need for <<scalability,replica shards>>,
+potentially halving the local storage needed to search your data.
+{search-snaps-cap} rely on the same snapshot mechanism you already use for
+backups and have minimal impact on your snapshot repository storage costs.
+
+[discrete]
+[[mount-options]]
+=== Mount options
+
+To search a snapshot, you must first mount it locally as an index. There are two
+mounting options, each with a different local storage footprint:
+
+[[full-copy]]
+Full copy::
+Loads a full, local copy of the snapshotted index's shards into the cluster. The
+cold tier uses this option by default.
```

Comment:
Does the hot tier also use this option by default? I was wondering what was used for searchable snapshots created during the hot phase.